### DISCUSSION TITLE
ScriptReference/PropertyDrawer-fieldInfo

### DISCUSSION THREAD

**Discussion by giscus** [Developer-Notes-Extension/Unity-Notes#55] [Notes]
# ScriptReference/PropertyDrawer-fieldInfo

https://docs.unity3d.com/ScriptReference/PropertyDrawer-fieldInfo.html

**Comment by vertxxyz** [comment]
If applied to a collection, the FieldInfo will refer to the collection itself and not the members, which is different to what the property drawer itself is drawing (ie. each member).
This is counter-intuitive, but of course there is no FieldInfo describing a member of a collection.
